exporter/exporterhelper/internal/batch_sender_test.go

```diff
@@ -263,16 +263,27 @@ func TestBatchSender_Disabled(t *testing.T) {
 			require.NotNil(t, be)
 			require.NoError(t, err)
 
-			require.NoError(t, be.Start(context.Background(), componenttest.NewNopHost()))
+			// Add explicit error checking for Start
+			if err := be.Start(context.Background(), componenttest.NewNopHost()); err != nil {
+				t.Fatalf("Failed to start exporter: %v", err)
+			}
```

Comment:
Just add a comment like:

```
require.NoError(t, be.Start(context.Background(), componenttest.NewNopHost()), "Failed to start exporter")
```

But I am not sure this helps, since you will have anyway the line number.